app/controllers/defence_requests_controller.rb

```diff
@@ -106,7 +106,7 @@ def queue
   private
 
   def associate_cco
```

Comment:
We should be able to delete this method now that associating a cco occurs in the transition strategies. Should have spotted that, sorry.